dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/alert/ProcessAlertManager.java

```diff
@@ -217,13 +216,7 @@ public void sendAlertWorkerToleranceFault(ProcessInstance processInstance, List<
     public void sendAlertProcessInstance(ProcessInstance processInstance,
                                          List<TaskInstance> taskInstances,
                                          ProjectUser projectUser) {
-
-        if (!isNeedToSendWarning(processInstance)) {
```

Comment:
Why remove this check?

Reply:
Because it has been judged before calling the method sendAlertProcessInstance

Reply:
It's better don't remove this, since this method mighe be execute by another where, you can remove this check in `WorkflowExecuteRunnable`

Reply:
done